clouddriver-aws/src/main/groovy/com/netflix/spinnaker/clouddriver/aws/deploy/handlers/LoadBalancerV2UpsertHandler.groovy

```diff
@@ -42,24 +42,31 @@ class LoadBalancerV2UpsertHandler {
   private static String modifyTargetGroupAttributes(AmazonElasticLoadBalancing loadBalancing, LoadBalancer loadBalancer, TargetGroup targetGroup, UpsertAmazonLoadBalancerV2Description.Attributes attributes, DeployDefaults deployDefaults) {
     def targetGroupAttributes = []
     if (attributes) {
-      Integer deregistrationDelay = [attributes.deregistrationDelay, deployDefaults?.loadBalancing?.deregistrationDelayDefault].findResult(Closure.IDENTITY)
-      if (deregistrationDelay != null) {
-        targetGroupAttributes.add(new TargetGroupAttribute(key: "deregistration_delay.timeout_seconds", value: deregistrationDelay.toString()))
-      }
-      if (loadBalancer.type == 'application') {
-        if (attributes.stickinessEnabled != null) {
-          targetGroupAttributes.add(new TargetGroupAttribute(key: "stickiness.enabled", value: attributes.stickinessEnabled.toString()))
+      if(TargetTypeEnum.Lambda.toString().equalsIgnoreCase(targetGroup.getTargetType()))
+      {
+        if (attributes.multiValueHeadersEnabled != null) {
+          targetGroupAttributes.add(new TargetGroupAttribute(key: "lambda.multi_value_headers.enabled", value: attributes.multiValueHeadersEnabled))
         }
```

Comment:
These conditionals can be collapsed to a single line: Checking if the target type is lambda and that the `multiValueHeadersEnabled` is set.

Reply:
The reason for this is, first "if" has "else", if we combine two "if" statements, then we have to replace "else" with "if". Please let me know your thoughts.
```
if(condition1){
       if(condition2){
        }
}else{
......
}
```